boards/st/nucleo_f722ze/nucleo_f722ze.dts

```diff
@@ -174,7 +179,7 @@ zephyr_udc0: &usbotg_fs {
 
 &rtc {
 	clocks = <&rcc STM32_CLOCK_BUS_APB1 0x10000000>,
-		<&rcc STM32_SRC_LSI RTC_SEL(2)>;
+		<&rcc STM32_SRC_LSE RTC_SEL(1)>;
```

Comment:
Why moving to LSE ?

Reply:
More precise.

Reply:
This should be mentioned in the commit message. Also we need to keep at least one test on the LSI (could be an overlay in the test)